### PR TITLE
Add initial terminology for Woodpecker

### PR DESCRIPTION
**Pull request by 6543**
https://github.com/woodpecker-ci/woodpecker/issues/745

### CODE REVIEW

File: docs/tools-terminology.md
Comment (anbraten) [inline docs/tools-terminology.md:312]:
```suggestion
Some of the core Woodpecker terms are listed below. [[22]]
```

File: docs/tools-terminology.md
Comment (anbraten) [inline docs/tools-terminology.md:314]:
```suggestion
- **Pipeline**: Triggered by events like webhooks, crons or manual events. It contains one or more workflows.
```

File: docs/tools-terminology.md
Comment (anbraten) [inline docs/tools-terminology.md:315]:
```suggestion
- **Workflow**: Represents a sequential list of steps which are executed as a task by an agent. The result of a workflow is reported back to the forge.
```

Comment (6543) [reply docs/tools-terminology.md:315]: I called it "SCM Tools" instead of forge as it's used to mention gitlab,github,... in this doc - not sure what is best

**Review by justinabrahms — APPROVED** [review]

File: docs/tools-terminology.md
Comment (justinabrahms) [reply docs/tools-terminology.md:315]:
```suggestion
- **Workflow**: Represents a sequential list of steps which are executed as a task by an agent. The results of the workflow reports the result back to SCM Tools.
```

Comment (justinabrahms) [reply docs/tools-terminology.md:315]: ^^ Will merge after that change lands.

**Comment by 6543** [comment]
@justinabrahms thansk for merging :)

I guess your last suggestion didn't made it in ... should I create a new pull?

**Comment by justinabrahms** [comment]
No, I made the change myself and merged it. Thanks!

**Pull request merged**
